spotseeker_server/org_filters/uw_search.py

```diff
@@ -35,6 +36,14 @@ def filter_query(self, query):
                 spotextendedinfo__key="app_type"
                 )
 
+        if 'extended_info:group' in self.request.GET:
```

Comment:
Ok, with this implementation I see why you were targeting the uw_search. I think this needs to be 'extended_info:uwgroup' though. @devights: do you agree?